website/www/site/content/en/contribute/_index.md

```diff
@@ -321,69 +367,42 @@ export GOLANG_PROTOBUF_REGISTRATION_CONFLICT=ignore
 actionable comments for 60 days.  Author of a closed pull request is welcome to
 reopen the same pull request again in the future. The associated JIRAs will be
 unassigned from the author but will stay open.
 
-## Accounts and Permissions
-
-- [Beam issue tracker (JIRA)](https://issues.apache.org/jira/projects/BEAM/issues):
-  Anyone can access it and browse issues. Anyone can register an account and login
-  to create issues or add comments. Only contributors can be assigned issues. If
-  you want to be assigned issues, a PMC member can add you to the project contributor
-  group.  Email the [dev@ mailing list](/community/contact-us)
-  to ask to be added as a contributor in the Beam issue tracker, and include your ASF Jira username.
-
-- [Beam Wiki Space](https://cwiki.apache.org/confluence/display/BEAM/Apache+Beam):
-  Anyone has read access. If you wish to contribute changes, please create an account and request edit access on the
-  [dev@ mailing list](/community/contact-us) (include your Wiki account user ID).
-
-- Pull requests can only be merged by a
-  [Beam committer](https://home.apache.org/phonebook.html?pmc=beam).
-
-- [Voting on a release](https://www.apache.org/foundation/voting.html): Everyone can vote. Only
-  [Beam PMC](https://home.apache.org/phonebook.html?pmc=beam) members should mark their votes as binding.
+### Troubleshooting
 
-## Communication
+If you run into any issues, check out the [contribution FAQ](https://cwiki.apache.org/confluence/display/BEAM/Contributor+FAQ) or ask on the [dev@ mailing list](/community/contact-us/) or [#beam channel of the ASF slack](/community/contact-us/).
 
-All communication is expected to align with the [Code of Conduct](https://www.apache.org/foundation/policies/conduct).
+If you didn't find the information you were looking for in this guide, please
+[reach out to the Beam community](/community/contact-us).
 
-Discussion about contributing code to Beam happens on the [dev@ mailing list](/community/contact-us/). Introduce yourself!
+</div>
 
-Questions can be asked on the [#beam channel of the ASF slack](/community/contact-us/). Introduce yourself!
+## Find Efforts to Contribute to
+A great way to contribute is to join an existing effort. If you want to get involved but don’t have a project in mind, check our [list of open starter tasks](https://s.apache.org/beam-starter-tasks).
+For the most intensive efforts, check out the [roadmap](https://beam.apache.org/roadmap/).
 
-## Additional resources
+## Additional Resources
+Please see Beam developers’ Wiki Contributor FAQ for more information.
 
-If you are contributing a `PTransform` to Beam, we have an extensive
-[PTransform Style Guide](/contribute/ptransform-style-guide).
+If you are contributing a [PTransform](https://s.apache.org/beam-starter-tasks) to Beam, we have an extensive [PTransform Style Guide](https://beam.apache.org/contribute/ptransform-style-guide).
```

Comment:
Unlink "PTransform" - should be code-style  
![image](https://user-images.githubusercontent.com/99457894/159712874-2820bfd1-b475-450a-8124-e2ae0f7cfd16.png)

Reply:
done